src/main/java/net/pms/encoders/FFmpegAudio.java

```diff
@@ -247,6 +272,44 @@ public synchronized ProcessWrapper launchTranscode(
 		return pw;
 	}
 
+	/**
```

Comment:
@Sami32 I just checked and those `parseOptions` methods are identical to the ones in `FFmpegVideo` except for comment changes in the second one, so it would be better to remove those from this file and if you want, make the comment changes in `FFmpegVideo`. Otherwise it's just writing the same code twice which also means bugfixes need to get written twice, etc.